src/3p-frame.js

```diff
@@ -135,7 +142,7 @@ export function getIframe(parentWindow, parentElement, opt_type, opt_context) {
     this.readyState = 'complete';
   };
   iframe.setAttribute(
-      'data-amp-3p-sentinel', attributes._context.amp3pSentinel);
+      'data-amp-3p-sentinel', attributes._context.sentinel);
```

Comment:
should we just generate the sentinel here, if this is the only place uses the sentinel.